bindings/rabbitmq/rabbitmq.go

```diff
@@ -34,7 +35,9 @@ type rabbitMQMetadata struct {
 	Host             string `json:"host"`
 	Durable          bool   `json:"durable,string"`
 	DeleteWhenUnused bool   `json:"deleteWhenUnused,string"`
+	PrefetchCount    string `json:"prefetchCount"`
```

Comment:
Make "PrefetchCount    string `json:"prefetchCount"`" into PrefetchCount    int `json:"prefetchCount"` and remove `prefetchCount    int`

Reply:
It seems JSON.Unmarshal Couldn't resolve the string to int because metadata is map[string]string.
`&json.UnmarshalTypeError{Value:"string", Type:(*reflect.rtype)(0x1236720), Offset:84, Struct:"rabbitMQMetadata", Field:"prefetchCount"}`

Reply:
![2C3928DC421EE14E4530CA9AAEABFE73](https://user-images.githubusercontent.com/32089134/92358624-58d5b400-f11c-11ea-8990-dbb9ac0dd0bd.png)
 I check the source code of JSON, and unless we change the int to other, It may not be Unmarshal success.

Reply:
I update the way to use metadate["prefetch"] to retrieve the data instead of the JSON decode now.